src/PostRector/Rector/ClassRenamingPostRector.php

```diff
@@ -23,6 +23,11 @@ final class ClassRenamingPostRector extends AbstractPostRector
 {
     private FileWithoutNamespace|Namespace_|null $rootNode = null;
 
+    /**
+     * @var array<string, string>
+     */
+    private array $oldToNewClasses;
```

Comment:
better to have default empty array to avoid invalid usage as not initiliazed by `__construct()`

```suggestion
    private array $oldToNewClasses = [];
```